src/runtime/data/kata-collect-data.sh

```diff
@@ -89,11 +89,14 @@ Usage: $script_name [options]
 Summary: Collect data about an installation of $project_name.
 
 Description: Run this script as root to obtain a markdown-formatted summary
-  of the environment of the $PROJECT_NAME installation. The output of this script
+  of the environment of the $project_name installation. The output of this script
   can be pasted directly into a github issue at the address below:
 
       $issue_url
 
+Commands:
```

Comment:
You will need to adjust the usage header from `Usage: $script_name [options]` to `Usage: $script_name [command] [options]`